<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDriver</class>
 <widget class="QWidget" name="FrmDriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drivers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Raw Stream Recorder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="m_edRec"/>
        </item>
        <item>
         <widget class="QToolButton" name="m_btnRec">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="m_ckbBinRecWrite">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="m_tblDrivers"/>
   </item>
   <item row="2" column="0">
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="m_btnNew">
     <property name="text">
      <string>New Driver</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="m_btnDelete">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete Driver</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_edRec</tabstop>
  <tabstop>m_btnRec</tabstop>
  <tabstop>m_ckbBinRecWrite</tabstop>
  <tabstop>m_tblDrivers</tabstop>
  <tabstop>m_btnNew</tabstop>
  <tabstop>m_btnDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
